File: Organizer.Model/Entities/Card.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Organizer.Model
{
    [Table("Card")]
    public class Card
    {
        [Key]
        [Column("CardId")]
        public long Id { get; set; }

        public CardType Type { get; set; }

        public long CompanyId { get; set; }

        public CardCompany Company { get; set; }

        public string Number { get; set; }

        public string Owner { get; set; }

        public DateTime ExpirationDate { get; set; }
    }
}

File: Organizer.Loader/src/entities/card.ts
import { CardType } from "entities/cardType";
import { CardCompany } from "entities/cardCompany";

export class Card {
    public Id: number;
    public Type: CardType;
    public CompanyId: number;
    public Number: string;
    public Owner: string;
    public ExpirationDate: Date;
}